apps/pipelines/nodes/nodes.py

```diff
@@ -201,9 +207,18 @@ def _get_current_datetime(self, session):
 class SendEmail(PipelineNode):
     __human_name__ = "Send an email"
     __node_description__ = ""
-    recipient_list: str
+    recipient_list: str = Field(description="A comma-separated list of email addresses")
     subject: str
 
+    @field_validator("recipient_list", mode="before")
+    def recipient_list_has_valid_emails(cls, value):
+        email_pattern = re.compile(r"^[\w\.-]+@[\w\.-]+\.\w+$")
+        emails = [email.strip() for email in value.split(",")]
+        for email in emails:
+            if not email_pattern.match(email):
```

Comment:
Django already has an email validator: `django.core.validators.validate_email`

Reply:
Nice, here it is: https://github.com/dimagi/open-chat-studio/pull/857/commits/a8631e6f0c0c070ddfa018e2b0c58bb6e1ee660a